src/univalue/univalue.h

```diff
@@ -26,6 +30,9 @@ class UniValue {
     UniValue(int64_t val_) {
         setInt(val_);
     }
+    UniValue(bool val_) {
+        setBool(val_);
+    }
```

Comment:
This PR also adds a clean boolean handling to UniValue (necessary for JSONRPC compatibility).